src/pagopa_proxy.tf

```diff
@@ -0,0 +1,147 @@
+resource "azurerm_resource_group" "pagopa_proxy_rg" {
+  count    = var.pagopa_proxy_enabled ? 1 : 0
+  name     = format("%s-pagopa-proxy-rg", local.project)
+  location = var.location
+
+  tags = var.tags
+}
+
+module "pagopa_proxy_snet" {
+  count                = var.pagopa_proxy_enabled && var.cidr_subnet_pagopa_proxy != null ? 1 : 0
+  source               = "git::https://github.com/pagopa/azurerm.git//subnet?ref=v2.0.19"
+  name                 = format("%s-pagopa-proxy-snet", local.project)
+  address_prefixes     = var.cidr_subnet_pagopa_proxy
+  resource_group_name  = azurerm_resource_group.rg_vnet.name
+  virtual_network_name = module.vnet.name
```

Comment:
how pagopa-proxy will contact pagopa-nodo apis?

Reply:
The goal is to use apim pagoPA - SIA connection, so (in another PR) we will map the nodo services on the apim (used from pagopa-proxy)